### ISSUE TITLE
When last item's state in projected effects is clicked, context menu is shown

### ISSUE THREAD

**Issue by DarkFenX** [pyfa-org/Pyfa#1926]
Pyfa master (2.8.0), linux, wx 4.0.4, gtk, wayland

Add 1 heatable module as TP to projected effects, right-click its state

Observed: module is heated, context menu is shown
Expected: module is heated, no context menu

Happens only for the last context menu in the list.

spawnMenu check fails:
```
if self.getColumn(self.ScreenToClient(event.Position)) == self.getColIndex(State)
```
It happens because `self.HitTest(pt)` which is used down the stack returns `rowIndex = -1` and `flags = 0`. Most likely an upstream issue.

**Comment by DarkFenX** [comment]
Affects xorg as well.

**Comment by DarkFenX** [comment]
Apparently it happens because HitTest() doesn't take into consideration optional header row.

https://github.com/wxWidgets/Phoenix/issues/232
http://trac.wxwidgets.org/ticket/17561

**Comment by DarkFenX** [comment]
As a note: I've added method `getRow()` to `display.py` which works around referred issue. Code which uses said method also has workarounds (to provide sane fallback when user clicks last rows). If that method is fixed, all such code should be reworked to rely on clicked item instead.

**Comment by DarkFenX** [comment]
Fixed by workaround in abe015bee39c674a9dc343135050e8a720fed3e3
